NOTICE

```diff
@@ -1 +1 @@
-Copyright (c) 2017-2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+Copyright Pravega Authors.
```

Comment:
@fpj @derekm  Should we leave the dell copyright as pravega repo here? https://github.com/pravega/pravega/blob/master/NOTICE#L2

Reply:
We have replaced the all in the `pravega/pravega` repo, and the idea is indeed that we replace them across all repositories. @derekm did say that he would create issues across repositories so that we could change it across the org. @derekm?

Reply:
Yeah, we should preserve old copyrights in the NOTICE file and also include years. For example:
https://github.com/pravega/pravega/blob/master/NOTICE

In file headers, we have permission from Dell Legal to replace old copyrights. For example:
https://github.com/pravega/pravega/blob/master/segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentContainerManager.java#L2